kvm-bindings/src/x86_64/bindings.rs

```diff
@@ -857,10 +857,7 @@ pub struct kvm_ioapic_state {
 }
 #[repr(C)]
 #[derive(Copy, Clone)]
-#[cfg_attr(
-    feature = "serde",
-    derive(zerocopy::AsBytes, zerocopy::FromBytes, zerocopy::FromZeroes)
-)]
+#[cfg_attr(feature = "serde", derive(zerocopy::Immutable, zerocopy::FromBytes))]
```

Comment:
mhh, why no `IntoBytes` for this one? (I guess it compiles so we don't we need it, but I wonder why haha)